### ISSUE TITLE
Unable to ignore comments, adding indents to empty commented lines

### ISSUE THREAD

**Issue by nickdomenicali** [nicklockwood/SwiftFormat#209]
When commenting out a block of code the empty lines that are also commented get indented to the level of scope.

for example:
```swift
class Bar {
    class Deeper {
//        class Foo {
//
//        }
    }
}
```
both `swiftformat .` and `swiftformat . --comments ignore` indent the empty comment line
```swift
class Bar {
    class Deeper {
//        class Foo {
        //
//        }
    }
}
```

**Comment by nicklockwood** [comment]
@nickdomenicali @hyperknot sorry I took so long to get to this. I've found the problem and it will be fixed in the next release.

**Comment by nicklockwood** [comment]
Fixed in 0.33.0
